HW9/OlehShynkaruk/task_1/main.py

```diff
@@ -0,0 +1,29 @@
+import random
+
+random_number = random.randint(1, 100)
+
+attempts = 10
+
+print("Welcome to the Guessing Game!")
+print("I have randomly chosen a number between 1 and 100.")
+print(f"You have {attempts} attempts to guess it. Good luck!")
+
+for i in range(attempts):
+    attempts_left = attempts - 1
+    
+    try:
+        guess = int(input(f"Attempt {i+1}/{attempts}. Enter your guess: "))
+    except Error:
+            print("Please enter a valid number.")
+            continue
+    
+    if guess == random_number:
+        print(f"Congratulations! You've guessed the number {random_number} correctly!")
+        break
+    elif guess < random_number:
```

Comment:
After a break statement, there's no need to use elif - a simple if statement is sufficient since the previous conditions have already been exited.